server/app/app.js

```diff
@@ -0,0 +1,142 @@
+const express = require('express');
+const Promise = require('bluebird');
+var cors = require('cors');
+const app = express();
+const { WishList, WishListItem } = require('../models');
+const errorHandler = require('../middleware/errorHandler');
+
+app.use(express.json());
+app.use(errorHandler);
+app.use(cors());
+
+const getAllWishlists = async () => {
+  return WishList.fetchAll({ withRelated: ['items'], require: true });
+};
+
+const getWishlistById = async (id) => {
+  return WishList.where('id', '=', id).fetch({
+    withRelated: ['items'],
+    require: true,
+  });
+};
+
+const createWishlist = async ({ name, author, items }) => {
+  return new WishList({ name, author })
+    .save(null, {
+      require: true,
+      method: 'insert',
+    })
+    .then(({ id }) => {
+      return Promise.all(
+        items.map((item) => {
+          return new WishListItem({ name: item, wishlist_id: id }).save(null, {
+            require: true,
+            method: 'insert',
+          });
+        })
+      );
+    });
+};
+
+const deleteWishlist = async (id) => {
+  return new WishList({ id }).destroy({ require: true });
+};
+
+const updateWishlist = async (id, { name, author }) => {
+  return new WishList({ id, name, author }).save(null, {
+    require: true,
+    method: 'update',
+  });
+};
+
+const updateItem = async (id, { name }) => {
+  return new WishListItem({ id }).save({ name }, {
+    require: true,
+    method: 'update',
+    patch: true,
+  });
+};
+
+const deleteItem = async (id) => {
+  return new WishListItem({ id }).destroy({ require: true });
+};
```

Comment:
It might be helpful to move these functionalities to a separate file and import them into the server for use in the routes (i.e. "separate the concerns" of Database Manipulation from Server Route Functionality into different files).  

The actual content of the functions could remain essentially the same, we'd just be able to keep their definition out of the app.js file of the server - much like you've done with the models, for instance.